### PR TITLE
Bug/2575 Linkcheck Scanning isn't being enabled on content types when SEO Bundle is enabled

### PR DESCRIPTION
**Pull request by owenmorrill**

### CODE REVIEW

File: modules/features/cu_linkchecker/cu_linkchecker.install [hunk 1]
```diff
@@ -4,7 +4,13 @@
  * Implements hook_install().
  */
 function cu_linkchecker_install() {
+  $content_types = node_type_get_types();
```
Comment (alexfinnarn) [inline modules/features/cu_linkchecker/cu_linkchecker.install]: File and video content types were excluded in the original feature export.

File: modules/features/cu_linkchecker/cu_linkchecker.module [hunk 2]
```diff
@@ -48,3 +48,24 @@ function cu_linkchecker_analyze_form_submit($form, $form_state) {
   variable_set('cu_linkchecker_status', 1);
   drupal_set_message('The site\'s content has been analyzed for links and the links have been added to the link checker queue.');
 }
+
+/**
+ * Implements hook_cron().
+ *
+ * Every 30 days, check to see if new content types have been enabled.
```
Comment (alexfinnarn) [inline modules/features/cu_linkchecker/cu_linkchecker.module]: Why is this needed every month?

**Comment by owenmorrill** [comment]
#2575
